docs/tools/automated-testing/overview.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Automated Testing
+layout: en
+permalink: docs/tools/automated-testing/overview/
+---
+
+Running an automated test for your mobile apps – native, hybrid or web Application is better than a manual test in some cases, since it doesn't rely on human resources in order to be run, hence it is cheaper, more relaiable and more accurate than the manual testing.
+Automated testing can be run using many testing platforms (e.g. [JUnit](http://junit.org/junit4/), [TestNG](http://testng.org/doc/index.html), [WebDriverIO](http://webdriver.io/) ...) in many programming languages such as Java, Ruby, Javascript, PHP, Python and more. This basically means that it is easy to anyone to write an automated test.
+
+Here is a list of the test automation frameworks we support:
+
+###[Appium](/docs/tools/appium/introduction/)
+<a href="http://appium.io/" target"blank">Appium</a> is an open-source test-automation framework for mobile apps – native, hybrid and web apps are supported. It drives iOS and Android apps using the WebDriver JSON wire protocol. Selenium also uses the JSON wire protocol. If you are familiar with Selenium for web testing, Appium will definitely be easy to get started with. Accessing the source code of the tested app is not required at all for using Appium. [Start testing now.](#ten_minutes)
+
+###[Espresso](/docs/testing-tools/robotium-espresso/setup/)
+<a href="https://google.github.io/android-testing-support-library/docs/espresso/index.html" target"blank">Espresso</a> is a test automation framework for Android applications developed by Google. It enables you to do automatic black-box UI tests. Therefor it uses the <a href="http://developer.android.com/reference/android/app/Instrumentation.html" target="_blank">Android instrumentation</a> to inspect and interact with the activities you are testing. To locate the UI elements it draws back on <a href="https://github.com/hamcrest" target="_blank">hamcrest matchers</a>. Accessing the source code of the tested app is required to use Espresso. [Learn more](/docs/testing-tools/robotium-espresso/setup/)
```

Comment:
"to **run** black-box UI tests." <- always use a specific term rather than a very general one (like "do") if you can, it conveys more information and makes it sound like you know what you are talking about, as opposed making things sound very vague